src/api.js

```diff
@@ -756,11 +762,39 @@ API.prototype.router = function(options) {
         component:  options.component
       }));
     }
+    if (monitor) {
+      middleware.push( (req, res, next) => {
```

Comment:
Make this a function that returns a  function like all the other middle ware things.... why not?